packages/sdk/src/websocket_decompress_adapter.ts

```diff
@@ -107,6 +107,12 @@ export class WebsocketDecompressAdapter {
     }
 
     const databaseUrl = new URL(`v1/database/${nameOrAddress}/subscribe`, url);
+    if (url.searchParams.has('token')) {
```

Comment:
I would prefer to not use `searchParams` for this. I have an alternative [here](https://github.com/clockworklabs/spacetimedb-typescript-sdk/commit/c67e88ab87f23a465a4feba878aa21c6e55f9f38) which I think is more straightforward.